README.md

```diff
@@ -97,8 +97,8 @@ sudo ansible-playbook -i inventory/hosts.localhost playbooks/deploy_cluster.yml
 
 ## Complete Production Installation Documentation:
 
-- [OpenShift Enterprise](https://docs.openshift.com/enterprise/latest/install_config/install/advanced_install.html)
-- [OpenShift Origin](https://docs.openshift.org/latest/install_config/install/advanced_install.html)
+- [OpenShift Container Platform](https://docs.openshift.com/container-platform/3.9/install_config/install/planning.html)
+- [OpenShift Origin](https://docs.okd.io/3.9/install_config/install/planning.html)
```

Comment:
You didn't change the anchor text to OKD like you did in the other PRs

Reply:
@patrickdillon That was on purpose, because the renaming to OKD was with 3.10. But I can change it OKD if you want.